linker/Tests/Mono.Linker.Tests.Cases.Expectations/Metadata/KeepTypeForwarderOnlyAssembliesAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Metadata {
+	[AttributeUsage (AttributeTargets.Class)]
+	public sealed class KeepTypeForwarderOnlyAssembliesAttribute : BaseMetadataAttribute {
```

Comment:
I think it's be easier to have one attribute with enum value for all boolean like settings instead of X individual attributes